include/libtorrent/torrent_info.hpp

```diff
@@ -450,37 +452,36 @@ namespace libtorrent
 
 		// ``merkle_tree()`` returns a reference to the merkle tree for this
 		// torrent, if any.
-		//
+		std::vector<sha1_hash> const& merkle_tree() const { return m_merkle_tree; }
+
 		// ``set_merkle_tree()`` moves the passed in merkle tree into the
 		// torrent_info object. i.e. ``h`` will not be identical after the call.
 		// You need to set the merkle tree for a torrent that you've just created
 		// (as a merkle torrent). The merkle tree is retrieved from the
 		// ``create_torrent::merkle_tree()`` function, and need to be saved
 		// separately from the torrent file itself. Once it's added to
 		// libtorrent, the merkle tree will be persisted in the resume data.
-		std::vector<sha1_hash> const& merkle_tree() const { return m_merkle_tree; }
 		void set_merkle_tree(std::vector<sha1_hash>& h)
 		{ TORRENT_ASSERT(h.size() == m_merkle_tree.size() ); m_merkle_tree.swap(h); }
 
 		// ``name()`` returns the name of the torrent.
-		//
-		// ``comment()`` returns the comment associated with the torrent. If
-		// there's no comment, it will return an empty string.
+		// Both the name and the comment is UTF-8 encoded strings.
```

Comment:
this statement would make more sense if ``name()`` and ``comment()`` were grouped together